packages/store/src/internal/state-factory.ts

```diff
@@ -218,6 +212,22 @@ export class StateFactory {
     return forkJoin(results);
   }
 
+  private addToStatesMap(stateClasses: StateClass[]): { newStates: StateClass[] } {
```

Comment:
{ newStates: StateClass[] } - create interface

Reply:
Not needed. This is only part of this one method's signature.